definitions/procedures/restore/configs.rb

```diff
@@ -56,7 +56,11 @@ def clean_conflicting_data
     def reset_qpid_jrnls
       # on restore without pulp data qpid fails to start
       # https://access.redhat.com/solutions/4645231
-      execute('rm -rf /var/lib/qpidd/.qpidd/qls/dat2/__db.00*')
+      ['/var/lib/qpidd/.qpidd/', '/var/lib/qpidd/'].each do |qpidd_path|
+        if File.exist?("#{qpidd_path}/qls/dat2/")
```

Comment:
I never know what's better, `File.exist?` or `Dir.exist?`

Reply:
if its a directory, it probably doesn't matter.  Dir.exists? might show better intent, but in code it likely doesn't matter